packages/yourloops/components/patient/add-patient-dialog.tsx

```diff
@@ -68,19 +68,13 @@ export const AddPatientDialog: FunctionComponent<AddDialogProps> = ({ onClose, o
   const isValidEmail = (mail = email): boolean => mail.length > 0 && REGEX_EMAIL.test(mail)
 
   const handleClickAdd = async (): Promise<void> => {
-    if (isValidEmail() && selectedTeam.id.length > 0) {
+    if (isValidEmail()) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏This check is useless too, the button is activated only when email is valid 😉‏